README.md

```diff
@@ -16,6 +16,114 @@ If you are a PG&E customer you can link your account now!  If you are not a PG&E
 
 ![Design](/docs/PGESMD_sketch_full.png)
 
+# Development
+
+## Environment Setup (Ubuntu 20.04)
+
+Process notes here: https://github.com/JPHutchins/open-energy-view/issues/31
+
+The following notes are for setting up the environment with a Windows 10 host and Ubuntu 20.04 on WSL2.  Please submit a PR if you find necessary adaptations on your environment.
+
+Personally I use VSCode from the Windows host utilizing the "Remote - SSH" and "Remote - WSL" extensions.
+
+### Clone this repository
+```
+git clone git@github.com:JPHutchins/open-energy-view.git
+cd open-energy-view
+git status
+```
+### Install backend dependencies
+* **Install python requirements**
+
+  Note: check your python3 version
+  ```
+  sudo apt install python3.8-venv build-essential python3-dev
+  ```
+* **Create the virtual environment and install packages**
+  ```
+  python3 -m venv venv
+  source venv/bin/activate
+  pip3 install -r requirements.txt
+  ```
+* **Install and configure rabbitmq**
+  * Install erlang:
+    ```
+    sudo apt update
+    sudo apt install software-properties-common apt-transport-https
+    wget -O- https://packages.erlang-solutions.com/ubuntu/erlang_solutions.asc | sudo apt-key add -
+    echo "deb https://packages.erlang-solutions.com/ubuntu focal contrib" | sudo tee /etc/apt/sources.list.d/rabbitmq.list
+    sudo apt update
+    sudo apt install erlang
+    ```
+  * Install rabbitmq
+    ```
+    curl -s https://packagecloud.io/install/repositories/rabbitmq/rabbitmq-server/script.deb.sh | sudo bash
+    sudo apt install rabbitmq-server
+  * Start rabbitmq-server
+    ```
+    sudo service rabbitmq-server start
+    sudo service rabbitmq-server status # verify that it is running
```

Comment:
Would suggest formatting this like the other steps rather than a comment:
  * Start rabbitmq-server
    ```
    sudo service rabbitmq-server status
    ```

  * Verify that rabbitmq-server is running
    ```
    sudo service rabbitmq-server status
    ```